src/binaryen.cpp

```diff
@@ -509,10 +509,13 @@ ExecutionResult BinaryenEngine::execute(
   return result;
 }
 
-void BinaryenEngine::loadModule(vector<uint8_t> const& code, wasm::Module & module)
+void BinaryenEngine::loadModule(bytes_view code, wasm::Module& module)
 {
   try {
-    wasm::WasmBinaryBuilder parser(module, reinterpret_cast<vector<char> const&>(code), false);
+    auto const b = reinterpret_cast<char const*>(code.begin());
+    auto const e = reinterpret_cast<char const*>(code.end());
+    std::vector<char> codeCopy{b, e};
```

Comment:
Is there no way to avoid the copy here just because change the type form `uint8_t` to `char`?

Reply:
No other than improving Binaryen API.

Reply:
Improving how? Binaryen already takes a reference.

Reply:
The problem is Binaryen forces you to pass a vector.